test/snapshots/no-duplicate-literal-elements.mjs.md

```diff
@@ -0,0 +1,329 @@
+# Snapshot report for `test/no-duplicate-literal-elements.mjs`
+
+The actual snapshot is saved in `no-duplicate-literal-elements.mjs.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## Invalid #1
+      1 | const foo = new Set([1, 2, 1])
+
+> Error 1/1
+
+    `␊
+    > 1 | const foo = new Set([1, 2, 1])␊
+        |                     ^^^^^^^^^ Hardcoded \`Set\` of \`1\` with duplicates.␊
```

Comment:
Can we just point to the duplicate item?

```text
    > 1 | const foo = new Set([1, 2, 1])␊
        |                            ^ A message says it's duplicated.
```

Reply:
In that case I have to check for duplicates in the selector stage.

I have no idea about this part.

Can you provide some direction?

Reply:
Select `ArrayExpression > Literal`, check if value exists in siblings before it.

Reply:
This will cause multiple messages on the same array, maybe we can simply change your current implementation to report on the first duplicated item.

You can use `.find()` to find the first one.

Reply:
Great!
I will try it.

Reply:
@fisker  just report first duplicate index now.